src/main/java/org/apache/commons/collections4/iterators/UnmodifiableListIterator.java

```diff
@@ -53,6 +53,14 @@
         return new UnmodifiableListIterator<>(iterator);
     }
 
+    /**
+     * use "org.apache.commons.collections4.iterators.UnmodifiableListIterator#unmodifiableListIterator" instead.
```

Comment:
Would need the @deprecated javadoc here too to help IDE's I think?

Reply:
Hi thanks for the suggestion, I added a commit for this.